build.gradle

```diff
@@ -203,6 +203,7 @@ project('segmentstore:server') {
         compile project(':segmentstore:contracts')
         compile project(':segmentstore:storage')
         compile project(':shared:metrics')
+        compile group: 'org.apache.commons', name: 'commons-lang3', version: commonsLang3ControllerContractVersion
```

Comment:
There is no need to add this dependency just for a `Pair`. Please remove this.

Reply:
@andreipaduroiu  Please see: https://github.com/pravega/pravega/pull/1842